docs/examples/interpolated-periods.py

```diff
@@ -0,0 +1,68 @@
+"""
+Interpolated Data Periods
+===================
```

Comment:
Underline length

Reply:
Updated in 
commit 839c794a37982d4b12951cbb283d33774f27bb03